x2i-data/apta.yaml

```diff
@@ -0,0 +1,993 @@
+---
+name: APTA
+help: 'https://web.archive.org/web/20230407205814/https://docs.google.com/spreadsheets/u/0/d/1vrO-PypNiL8SlYCGEmuVFfHdQl2daXkM9tXkek0iUNQ/htmlview'
+prefix: pt
+escape: \
+replacers:
+  - raw:
+      - \\(.)
+      - $1
+  - - "tS};-j"
+    - 𝼗
+  - - "dZ};-j"
+    - 𝼒
+  - - tS};-R
+    - 𝼜
+  - - dZ};-R
+    - 𝼙
+  - - "dd};"
+    - ʭ
+  - - "dz};"
+    - ʣ
+  - - "dZ};"
+    - ʤ
+  - - "dz;};"
+    - ꭦ
+  - - "dZ;};"
+    - ʥ
+  - - "fN};"
+    - ʩ
+  - - "ls};"
+    - ʪ
+  - - "lz};"
+    - ʫ
+  - - "ts};"
+    - ʦ
+  - - "tS};"
+    - ʧ
+  - - "ts;};"
+    - ꭧ
+  - - "tS;};"
+    - ʨ
+  - - "1-R"
+    - 𝼚
+  - - o-R
+    - 𝼛
+  - - "!-R"
+    - 𝼊
+  - - R-L
+    - ꭨ
+  - - N-j
+    - 𝼔
+  - - Z-j
+    - 𝼘
+  - - "$-j"
+    - 𝼓
+  - - R-j
+    - 𝼕
+  - - "4-j"
+    - 𝼖
+  - - i-R
+    - ᶖ
+  - - u-R
+    - ᶙ
+  - - e-R
+    - ᶒ
+  - - "@-R"
+    - ᶕ
+  - - E-R
+    - ᶓ
+  - - 3-R
+    - ᶔ
+  - - O-R
+    - ᶗ
+  - - a-R
+    - ᶏ
+  - - A-R
+    - ᶐ
+  - - m-L
+    - ᵯ
+  - - n-L
+    - ᵰ
+  - - p-L
+    - ᵱ
+  - - t-L
+    - ᵵ
+  - - b-L
+    - ᵬ
+  - - d-L
+    - ᵭ
+  - - f-L
+    - ᵮ
+  - - s-L
+    - ᵴ
+  - - z-L
+    - ᵶ
+  - - r-L
+    - ᵲ
+  - - "4-L"
+    - ᵳ
+  - - m-j
+    - ᶆ
+  - - n-j
+    - ᶇ
+  - - p-j
+    - ᶈ
+  - - ^t-j
+    - ᶵ
+  - - t-j
+    - ƫ
+  - - k-j
+    - ᶄ
+  - - b-j
+    - ᶀ
+  - - d-j
+    - ᶁ
+  - - g-j
+    - ᶃ
+  - - f-j
+    - ᶂ
+  - - s-j
+    - ᶊ
+  - - S-j
+    - ᶋ
+  - - x-j
+    - ᶍ
+  - - v-j
+    - ᶌ
+  - - z-j
+    - ᶎ
+  - - ^l-j
+    - ᶪ
+  - - l-j
+    - ᶅ
+  - - r-j
+    - ᶉ
+  - - S-R
+    - ᶘ
+  - - Z-R
+    - ᶚ
+  - raw:
+      - '\#?\|\.?\|'
+      - ‖
+  - - "ww};"
+    - ʬ
+  - - "#1;<"
+    - ꜑
+  - - "#2;<"
+    - ꜐
+  - - "#3;<"
+    - ꜏
+  - - "#4;<"
+    - ꜎
+  - - "#5;<"
+    - ꜍
+  - - "t;<"
+    - 𝼉
+  - - "#1<"
+    - ꜖
+  - - "#2<"
+    - ꜕
+  - - "#3<"
+    - ꜔
+  - - "#4<"
+    - ꜓
+  - - "#5<"
+    - ꜒
+  - - "^#0"
+    - ⁰
+  - - "^#1"
+    - ¹
+  - - "^#2"
+    - ²
+  - - "^#3"
+    - ³
+  - - "^#4"
+    - ⁴
+  - - "^#5"
+    - ⁵
+  - - "^#6"
+    - ⁶
+  - - "^#7"
+    - ⁷
+  - - "^#8"
+    - ⁸
+  - - "^#9"
+    - ⁹
+  - - "^#;"
+    - 𐞷
+  - - -6|
+    - ̗ 
+  - - -9|
+    - ̖ 
+  - - +1|
+    - ̱ 
+  - - +5|
+    - ̄
+  - - +8|
+    - ́ 
+  - - +B|
+    - ̀ 
+  - - +C|
+    - ̂
+  - - +D|
+    - ̌
+  - - "#1|"
+    - ˍ     
+  - - "#3|"
+    - ˗
+  - - "#5|"
+    - ˉ
+  - - "#6|"
+    - ˏ
+  - - "#8|"
+    - ˊ
+  - - "#9|"
+    - ˎ
+  - - "#B|"
+    - ˎ
+  - - "#C|"
+    - ˆ
+  - - "#D|"
+    - ˇ
+  - - "#';"
+    - ˉ
+  - - "#1;"
+    - ꜌
+  - - "#2;"
+    - ꜋
+  - - "#3;"
+    - ꜊
+  - - "#4;"
+    - ꜉
+  - - "#5;"
+    - ꜈
+  - - "$;|"
+    - 𝼅
+  - - "1;|"
+    - ʅ
+  - - "4;|"
+    - 𝼈
+  - - "F;|"
+    - 𝼌
+  - - "I;|"
+    - ᵼ
+  - - "L;|"
+    - 𝼄
+  - - "N;|"
+    - 𝼇

```

Comment:
Same rendering issue here

Reply:
I see. For the ones that have no replacement, I guess it's fine to leave the unsupported character in for now.